1-WebApp-OIDC/1-5-B2C/README.md

```diff
@@ -0,0 +1,172 @@
+---
+services: active-directory
+platforms: dotnet
+author: TiagoBrenck
+level: 100
+client: ASP.NET Core Web App
+endpoint: AAD v2.0
+---
+# An ASP.NET Core Web app signing-in users with the Microsoft identity platform in Azure AD B2C
+
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=819)
+
+## Scenario
+
+This sample shows how to build a .NET Core 2.2 MVC Web app that uses OpenID Connect to sign in users in **Azure AD B2C**. It assumes you have some familiarity with **Azure AD B2C**. If you'd like to learn all that B2C has to offer, start with our documentation at aka.ms/aadb2c.
+
+![Sign in with Azure AD](ReadmeFiles/sign-in.png)
+
+## How to run this sample
+
+To run this sample:
+
+> Pre-requisites: Install .NET Core 2.2 or later (for example for Windows) by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```powershell
+git clone https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2.git
+```
+
+> Given that the name of the sample is very long, and so are the name of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
+
+### Step 2: Get your own Azure AD B2C tenant
+
+If you don't an Azure AD B2C tenant yet, you'll need to create an Azure AD B2C tenant by following [these instructions](https://azure.microsoft.com/documentation/articles/active-directory-b2c-get-started).
```

Comment:
>instructions [](start = 106, length = 12)

nit: consider expanding on what is happening here, so they can decide to click the link or not..ex "...tenant by following [these instructions on creating an AAD B2C tenant]()